scripts/end-to-end/remove-testcases.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
```

Comment:
Can you move these scripts into `scripts/endToEndExtraction` or something similar?